src/main/java/uk/gov/ons/ssdc/caseprocessor/messaging/NewCaseReceiver.java

```diff
@@ -45,43 +46,46 @@ public SampleReceiver(
   }
 
   @Transactional
-  @ServiceActivator(inputChannel = "sampleInputChannel", adviceChain = "retryAdvice")
-  public void receiveSample(Message<byte[]> message) {
-    Sample sample = convertJsonBytesToObject(message.getPayload(), Sample.class);
+  @ServiceActivator(inputChannel = "newCaseInputChannel", adviceChain = "retryAdvice")
+  public void receiveNewCase(Message<byte[]> message) {
+    EventDTO event = convertJsonBytesToEvent(message.getPayload());
 
-    if (caseRepository.existsById(sample.getCaseId())) {
+    NewCase newCasePayload = event.getPayload().getNewCase();
+
+    if (caseRepository.existsById(newCasePayload.getCaseId())) {
       // Case already exists, so let's not overwrite it... swallow the message quietly
       return;
     }
 
     OffsetDateTime messageTimestamp = getMsgTimeStamp(message);
 
     Optional<CollectionExercise> collexOpt =
-        collectionExerciseRepository.findById(sample.getCollectionExerciseId());
+        collectionExerciseRepository.findById(newCasePayload.getCollectionExerciseId());
 
     if (!collexOpt.isPresent()) {
       throw new RuntimeException(
-          "Collection exercise '" + sample.getCollectionExerciseId() + "' not found");
+          "Collection exercise '" + newCasePayload.getCollectionExerciseId() + "' not found");
     }
 
     CollectionExercise collex = collexOpt.get();
 
     Case newCase = new Case();
-    newCase.setId(sample.getCaseId());
+    newCase.setId(newCasePayload.getCaseId());
     newCase.setCollectionExercise(collex);
-    newCase.setSample(sample.getSample());
-    newCase.setSampleSensitive(sample.getSampleSensitive());
+    newCase.setSample(newCasePayload.getSample());
+    newCase.setSampleSensitive(newCasePayload.getSampleSensitive());
 
     newCase = saveNewCaseAndStampCaseRef(newCase);
-    caseService.emitCaseUpdate(newCase, sample.getJobId(), sample.getOriginatingUser());
+    caseService.emitCaseUpdate(
+        newCase, newCasePayload.getJobId(), newCasePayload.getOriginatingUser());
 
     eventLogger.logCaseEvent(
         newCase,
-        OffsetDateTime.now(),
-        "New case created from sample load",
+        event.getHeader().getDateTime(),
+        "New case created",
         EventType.NEW_CASE,
-        createEventDTO(TOPIC_SAMPLE, sample.getJobId(), sample.getOriginatingUser()),
-        RedactHelper.redact(sample),
+        event.getHeader(),
+        RedactHelper.redact(newCasePayload),
```

Comment:
no need to call the redact helper... `logCaseEvent` will do that for you already